matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/permalinks/PermalinkData.kt

```diff
@@ -54,7 +54,5 @@ sealed class PermalinkData {
 
     data class UserLink(val userId: String) : PermalinkData()
 
-    data class GroupLink(val groupId: String) : PermalinkData()
-
-    data class FallbackLink(val uri: Uri) : PermalinkData()
+    data class FallbackLink(val uri: Uri, val isLegacyGroupLink: Boolean = false) : PermalinkData()
```

Comment:
I've added a param here instead of keeping `GroupLink` to not force every `when` which will handle `PermalinkData` to explicitly handle `GroupLink` class. Since it's legacy I'd like to reduce impact on future development, but I'm not sure that making it param really helps, so I'd like to hear your thoughts